package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "cypress-training",
+  "version": "1.0.0",
+  "description": "This is a Workshop about Cypress",
+  "main": "index.js",
+  "scripts": {
+    "test": "cypress open"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/DollyHernandez/cypress-training.git"
+  },
+  "keywords": [
+    "ui-testing"
+  ],
+  "author": "Dolly Hernandez <dollyhernandezGit@gmail.com> (\u001b[C\u001b[DDollyHernandez)",
```

Comment:
Please change `\u001b[C\u001b[DDollyHernandez` by `(DollyHernandez)`

Reply:
Holmes, I updated the package.json